src/library/coverart.h

```diff
@@ -59,14 +70,52 @@ class CoverInfoRelative {
         *this = CoverInfoRelative();
     }
 
+    void setImage(
+            const QImage& image = QImage());
+    void setImageDigest(
+            QByteArray imageDigest,
+            quint16 legacyHash = defaultLegacyHash()) {
+        m_imageDigest = imageDigest;
+        m_legacyHash = legacyHash;
+    }
+
+    bool hasImage() const {
+        return imageHash() != CoverImageUtils::defaultHash();
+    }
+
+    const QByteArray imageDigest() const {
+        return m_imageDigest;
+    }
+    mixxx::cache_key_t imageHash() const {
```

Comment:
Should we rename this to `cacheKey`? I was a bit confused what hash this was referring to and since this returns a cache key type I think this would make more sense.